commcare_connect/opportunity/api/views.py

```diff
@@ -125,3 +127,18 @@ def post(self, *args, **kwargs):
         payment.confirmation_date = now()
         payment.save()
         return Response(status=200)
+
+
+class SMSStatusCallbackView(APIView):
```

Comment:
nit: i don't think this belongs in opportunity API views, since this is currently only for the api the mobile uses to update its backend. I think this could just be in the normal views folder, either in users or opportunities, although i would lean toward users because that is where the other invite related views are.

Reply:
Fixed. https://github.com/dimagi/commcare-connect/pull/305/commits/1867c9ec34dbf50ff8a76c88051d3df054529e07